### PR TITLE
[Tests] basic save hook tests

### PR DESCRIPTION
**Pull request by sayakpaul**
Adds basic save hook manager tests to ensure we have desired attributes correctly set. 

@bghira if this looks good to you, I will add it for other models and have it ready for review.

### CODE REVIEW

**Comment by bghira** [comment]
ah, cool. thank you - the test suite is underutilised. this is a good idea.

**Comment by sayakpaul** [comment]
I will polish it up then and make it ready for a good-first review. Thanks!

**Comment by sayakpaul** [comment]
@bghira ready for a review.

File: helpers/training/save_hooks.py [hunk 1]
```diff
@@ -1,6 +1,5 @@
 from diffusers.training_utils import EMAModel, _set_state_dict_into_text_encoder
 from helpers.training.wrappers import unwrap_model
-from diffusers.loaders import LoraLoaderMixin
```
Comment (sayakpaul) [inline helpers/training/save_hooks.py:3]: Unused import.

**Review by bghira — APPROVED** [review]
wow, really nice tests 💪

**Pull request merged**